app/actions/ControlActions.js

```diff
@@ -485,6 +485,28 @@ export const revokeTicketsAttempt = (passphrase) => async (
   }
 };
 
+export const DISCOVERUSAGE_ATTEMPT = "DISCOVERUSAGE_ATTEMPT";
+export const DISCOVERUSAGE_FAILED = "DISCOVERUSAGE_FAILED";
+export const DISCOVERUSAGE_SUCCESS = "DISCOVERUSAGE_SUCCESS";
+
+export const discoverUsageAttempt = (gapLimit) => async (
+  dispatch,
+  getState
+) => {
+  dispatch({ type: DISCOVERUSAGE_ATTEMPT });
+  const walletService = sel.walletService(getState());
+  try {
+    const discoverUsageResponse = await wallet.discoverUsage(
+      walletService,
+      gapLimit
+    );
+    dispatch({ discoverUsageResponse, type: DISCOVERUSAGE_SUCCESS });
+    dispatch(rescanAttempt(0));
+  } catch (error) {
+    dispatch({ error, type: DISCOVERUSAGE_FAILED });
```

Comment:
`DISCOVERUSAGE_FAILED` needs to be added to the snackbar, otherwise users don't get any feedback on errors.